tests/test_messages.py

```diff
@@ -27,6 +28,54 @@ def test_balance_proof(get_random_bp):
         bp.token_network_address = '0x11e14d102DA61F1a5cA36cfa96C3B831332357b4'
 
 
+def test_balance_proof():
+    balance_proof = BalanceProof(
```

Comment:
wouldn't it be easier to use `get_random_balance_proof()` here?

Reply:
Then we would need to make sure that the balance data can be set to something or None. I think it's easier that way right now.